test/test_mail_helpers.py

```diff
@@ -373,6 +373,27 @@ def test_error_is_not_raised_on_to_emails_set_to_a_tuple(self):
         except:
             self.fail('Mail() raised an error on a tuple of strings')
 
+    def test_error_is_not_raised_on_to_emails_includes_bcc_cc(self):
+        from sendgrid.helpers.mail import (PlainTextContent, HtmlContent)
+        self.maxDiff = None
+        to_emails = [
+            To('test+to0@example.com', 'Example To Name 0'),
+            Bcc('test+bcc@example.com', 'Example Bcc Name 1'),
+            Cc('test+cc@example.com', 'Example Cc Name 2')
+        ]
+
+        try:
+            Mail(
+                from_email=From('test+from@example.com', 'Example From Name'),
+                to_emails=to_emails,
+                subject=Subject('Sending with SendGrid is Fun'),
+                plain_text_content=PlainTextContent(
+                    'and easy to do anywhere, even with Python'),
+                html_content=HtmlContent(
+                    '<strong>and easy to do anywhere, even with Python</strong>'))
+        except:
+            self.fail('Mail() raised an error on a tuple of strings')
```

Comment:
It's a list of emails actually. But rather just drop the exception handling here since the test fails for exceptions anyway.

Reply:
This is what I get for duplicating another test 🤦 . Updating this test and the other that went with the previous pr